keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -1782,6 +1782,15 @@ Alpha:
 - Adequate documentation for the changes
 - Minimum viable test cases mentioned in Test Plan section
 
+Beta:
```

Comment:
I'd like to see a discussion of 
 1. documenting metrics for inspection (on their way to a v1 metrics API, but they don't have to be there).  
 2. recommended alerts for the prometheus, somewhere like here https://github.com/coreos/kube-prometheus/blob/master/manifests/prometheus-rules.yaml
 3. eliminating rate limiting by some means.  Something like https://github.com/kubernetes/enhancements/pull/1629 could do.  (It's hard to prove otherwise).

Reply:
> documenting metrics for inspection (on their way to a v1 metrics API, but they don't have to be there).
> recommended alerts for the prometheus, somewhere like here https://github.com/coreos/kube-prometheus/blob/master/manifests/prometheus-rules.yaml

is there a common repo to place these prometheus integration and related docs? cluster admins will definitely find these docs helpful if they feel like understand the system better..

> eliminating rate limiting by some means. Something like #1629 could do. (It's hard to prove otherwise).

added this to blocking list. i need an approach to opt-out client-side rate-limitting

Reply:
Regarding item 1, there is already some documentation at https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#observability .  Is something more being requested here?